packages/manager/apps/web/client/app/hosting/multisite/git-integration/association/git-association.html

```diff
@@ -129,6 +140,31 @@ <h3 data-translate="hosting_multisite_git_association_configure_auto"></h3>
     </div>
 </form>
 
+<div class="mb-5">
+    <a
+        class="oui-link_icon row m-0 p-0"
+        target="_blank"
+        rel="noopener"
+        data-ng-href="{{ ::$ctrl.guide.url }}"
+    >
+        <span
+            data-translate="hosting_multisite_git_association_guide_configure_webhook"
+        ></span>
+        <span class="oui-icon oui-icon-arrow-right" aria-hidden="true"></span>
```

Comment:
```suggestion
        <span class="oui-icon oui-icon-external-link" aria-hidden="true"></span>
```

Reply:
seems arrow is required here cf "maquette" 

![image](https://github.com/ovh/manager/assets/8767496/c891b34c-f917-4bc5-ab57-d730e9c1d86f)